src/components/createTestCase/CreateTestCase.tsx

```diff
@@ -132,6 +133,17 @@ const INITIAL_VALUES = {
   groupPopulations: [],
 } as TestCase;
 
+interface PropTypes {
+  isActive?: boolean;
+}
+
+const MenuItemContainer = tw.ul`bg-transparent flex px-8`;
+const MenuItem = styled.li((props: PropTypes) => [
+  tw`relative h-2 mr-3 ml-3 mt-3 mb-0 not-italic font-rubik font-medium text-sm leading-4 text-gray`,
+  props.isActive &&
+    tw`bg-white h-2 text-black font-medium pb-6 border-solid border-b-4 border-blue-950 bg-opacity-75`,
```

Comment:
looks like some classes are redundant(already there in line 142): h-2, font-medium
border solid may not be needed. you have border 950.
does the opacity need?

Reply:
Height is required property for maintaining the size of tabs column(it is also mentioned in the styling provided in the figma). Yeah I'll remove the remaining classes